play/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
You can use https://www.toptal.com/developers/gitignore/ to generate this file that includes more files and supports Windows hidden files as we ll

Reply:
I've updated the file using gitignore.io (specifically `gi macos,windows,react >> .gitignore`) and double checked it against the `.gitignore` files for `host` and `central`. I don't see a specific support for windows hidden files in .gitignore.io but I've added in a couple lines to the bottom that follows: https://stackoverflow.com/questions/8021441/how-to-ignore-all-hidden-directories-files-recursively-in-a-git-repository

let me know if this is too much a blunt instrument for the task though